code/src/components/Survey.js

```diff
@@ -0,0 +1,215 @@
+import React, {useState} from 'react';
+
+import '../style/survey.css';
+import { Welcome } from './Welcome';
+import { Text } from './Text';
+import { Select } from './Select';
+import { Radio } from './Radio';
+//import { Checkbox } from './Checkbox';
+import { Rating } from './Rating';
+import { Button } from './Button';
+import { Summary } from './Summary';
+
+export const Survey = () => {
+const [question, setQuestion] = useState(0);
+const [showSummary, setShowSummary] = useState(false);
+const [name, setName] = useState("");
+const [location, setLocation] = useState("");
+const [ageGroup, setAgeGroup] = useState("");
+const [rating, setRating] = useState(0);
+//const [musicGenre, setMusicGenre] = useState(false);
+
+const nextQuestion = () => setQuestion(question +1);
+
+const handleNameChange = event => {
+  setName(event.target.value);
+};
+
+const handleLocationChange = event => {
+  setLocation(event.target.value);
+};
+
+const handleAgeChange = event => {
+  setAgeGroup(event.target.value);
+}
+
+const handleRatingChange = event => {
+  setRating(event.target.value);  
+}
+
+const cityArray = [
+  {
+    text:"Select city...",
+    value:"", 
+  },
+  {
+    text:"Austin",
+    value:"austin", 
+  },
+  {
+    text:"Barcelona",
+    value:"barcelona",
+  },
+  {
+    text:"Berlin",
+    value:"berlin",
+  },
+  {
+    text:"Chicago",
+    value:"chicago",
+  },
+  {
+    text:"Gothenburg", 
```

Comment:
I have an idea that this array is better inside the "Select" component to make the Survey neat. Not sure my idea is correct. How do you think?

Reply:
I went with an array so that I could show "Select city..." as the default and I didn't want it to have a value. Also in one of Maks lectures he advised to use lowercase for values, as this is processed by the browser/server.